portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,27 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.*;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  ArrayList<String> greetings=new ArrayList<String>();//Creating arraylist 
+  public DataServlet(){
+    greetings.add("hello");
+    greetings.add("whats up?");
+    greetings.add("nice to see you!");
+    }
+    
```

Comment:
Couple nitpicks for style:
1.  Space before and after the equal sign on line 28
2.  Space after the semi colon and before the comment in line 28
3.  Space after the `//` on line 28.  If this line, no longer fits within 80 chars, alternative would be to put the comment above the line it describes
4.  Space before the bracket in line 29
5.  Match the indentation of the closing bracket on line 33 with the indentation of the opening bracket on line 29 (in other words 2 spaces, since the opening line starting with `public` has 2 spaces.


These things doesn't affect your code's behavior/performance.  But these would be more for better clarity and readability.  It would be good to get into the habit early on;  it'll help a lot in the long run once you're working on bigger code bases.